app/views/collections/_sidebar.html.erb

```diff
@@ -1,62 +1,60 @@
+      <li><%= link_to t(".parent_collection"), collection_path(@collection.parent) %></li>
     <% else %>
-      <li><%= span_if_current ts("Subcollections (%{count})", key: 'dashboard', :count => @collection.children.count), collection_collections_path(@collection) %></li>
+      <li><%= span_if_current t(".subcollections", count: @collection.children.count), collection_collections_path(@collection) %></li>
     <% end %>
   </ul>
 
   <!-- challenge section of dash -->
   <% if @collection.challenge %>
-    <%= render :partial => "challenge/#{challenge_class_name(@collection)}/challenge_sidebar" %>
+    <%= render partial: "challenge/#{challenge_class_name(@collection)}/challenge_sidebar" %>
   <% end %>
 
   <h4 class="landmark heading">Contents</h4>
   <ul class="navigation actions">
-    <li><%= span_if_current ts("Fandoms (%{count})", key: 'dashboard', count: SearchCounts.fandom_count_for_collection(@collection)), collection_fandoms_path(@collection) %></li>
+    <li><%= span_if_current t(".fandoms", count: SearchCounts.fandom_count_for_collection(@collection)), collection_fandoms_path(@collection) %></li>
 
-    <li><%= span_if_current ts("Works (%{count})", key: 'dashboard', count: SearchCounts.work_count_for_collection(@collection)), collection_works_path(@collection) %></li>
+    <li><%= span_if_current t(".works", count: SearchCounts.work_count_for_collection(@collection)), collection_works_path(@collection) %></li>
 
-    <li><%= span_if_current ts("Bookmarked Items (%{count})", key: 'dashboard', count: SearchCounts.bookmarkable_count_for_collection(@collection)), collection_bookmarks_path(@collection) %></li>
+    <li><%= span_if_current t(".bookmarks", count: SearchCounts.bookmarkable_count_for_collection(@collection)), collection_bookmarks_path(@collection) %></li>
 
     <li>
       <% if controller.controller_name == 'collections' && controller.action_name == 'show' %>
-      <%= link_to 'Random Items', 
-        :url => collection_path(@collection, :show_random => true), :method => :get, :remote => true %>  
-      <% else %>  
-      <%= link_to 'Random Items', collection_path(@collection, :show_random => true) %>
+      <%= link_to "Random Items", url: collection_path(@collection, show_random: true), method: :get, remote: false %>
+      <% else %>
+      <%= link_to "Random Items", collection_path(@collection, show_random: true) %>
       <% end %>
     </li>
-    <li><%= span_if_current ts('People', key: 'dashboard'), collection_people_path(@collection) %></li>
-    <li><%= span_if_current ts('Tags', key: 'dashboard'), collection_tags_path(@collection) %></li>
-  </ul>  
+    <li><%= span_if_current t(".people"), collection_people_path(@collection) %></li>
+    <li><%= span_if_current t(".tags"), collection_tags_path(@collection) %></li>
+  </ul>
 
-  <% if @collection.user_is_maintainer?(current_user)%>
+  <% if @collection.user_is_maintainer?(current_user) %>
     <h4 class="landmark heading">Choices</h4>
```

Comment:
Please make this text translatable as well, I would suggest `t(".landmark.choices")`.